src/encode.cpp

```diff
@@ -391,7 +391,7 @@ operator<<(MessageBuffer& buffer, const Header& msg)
   return buffer;
 }
 
-MessageBuffer&
+static MessageBuffer&
```

Comment:
Why just these 2?

Reply:
Because those are the two for which clang-tidy complained about missing declarations.  If you want, we could declare them in `encode.h`, but it didn't seem necessary to expose them.